sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -215,6 +214,8 @@ export default defineComponent({
       };
       initialData.value = {
         ...applicationData.data,
+        howWillYouBeAttendingTheProgram:
```

Comment:
As we talked, lets have the offering display variable, expected to be displayed in the read-only input, also added here in preparation for the upcoming changes in the forms.